templates/zerver/help/add-a-custom-linkification-filter.md

```diff
@@ -0,0 +1,76 @@
+# Add a custom linkification filter
+
+If you are an administrator of a Zulip organization, you can add custom
+linkification filters to your organizations that automatically turn phrases in
```

Comment:
I'm not sure `organizations` should be plural here. You're using the singular noun later in the document.